source/_integrations/plum_lightpad.markdown

```diff
@@ -20,7 +22,8 @@ Configurable/Dimmable Wi-Fi Lightswitch
 
 ## Configuration
 
-To enable Plum Lightpad support, add the following lines to your `configuration.yaml` file:
+To use Plum Lightpad devices in your installation, add your Plum account from the integrations page.
```

Comment:
Please describe to the user what to do. See examples of other integrations in our documentation

Reply:
thanks, updated!